### PR TITLE
add feedback view + form

### PR DESCRIPTION
**Pull request by der-john**
So I've successfully added a form and a view with the help of https://docs.djangoproject.com/en/1.10/topics/forms/. Its URL currently is `localhost:8000/w/feedback/`.

@2e2a: please fill in a correct recipient address in `meinberlin/views.py`.

Do we have security concerns?

### CODE REVIEW

**Comment by der-john** [comment]
this is incomplete.

**Comment by 2e2a** [comment]
> @2e2a: please fill in a correct recipient address in meinberlin/views.py

Could we add a setting parameter for this?

File: meinberlin/views.py [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+            subject = 'meinberlin feedback'
+            qa1 = 'Was gefällt Ihnen an den Inhalten von mein.berlin.de?\n\n' + form.cleaned_data['answer1'] + '\n\n'
+            qa2 = 'Was missfällt Ihnen an den Inhalten von mein.berlin.de?\n\n' + form.cleaned_data['answer2'] + '\n\n'
+            qa3 = 'Was gefällt Ihnen an der technischen Umsetzung von mein.berlin.de?\n\n' + form.cleaned_data['answer3'] + '\n\n'
+            qa4 = 'Was missfällt Ihnen an der technischen Umsetzung von mein.berlin.de?\n\n' + form.cleaned_data['answer4'] + '\n\n'
+            message = qa1 + qa2 + qa3 + qa4
+            sender = form.cleaned_data['sender_address']
+            recipients = ['feedback@mein.berlin.de']
+
+            send_mail(subject, message, sender, recipients, fail_silently=False)
```
Comment (2e2a) [inline meinberlin/views.py]: Should not be send from sender email, but from the default sender set in the config.

File: meinberlin_wagtail/urls.py [hunk 2]
```diff
@@ -18,8 +18,9 @@
 
     url(r'^w/search/$', search_views.search, name='search'),
 
+    url(r'^w/feedback/$', meinberlin_views.feedback_view, name='feedback'),
```
Comment (der-john) [inline meinberlin_wagtail/urls.py:21]: great.

File: meinberlin/views.py [hunk 3]
```diff
@@ -17,12 +18,13 @@ class FeedbackForm(forms.Form):
     answer4 = forms.CharField(label='Was missfällt Ihnen an der technischen Umsetzung von mein.berlin.de?',
         max_length=1000,
         required=False)
+    sender_address = forms.EmailField(label='Wenn Sie eine Antwort bekommen möchten, geben Sie bitte Ihre email-Adresse an (optional):',
+        required=False)
```
Comment (der-john) [inline meinberlin/views.py:26]: thank you. I seem to have forgotten to push this.

Comment (2e2a) [reply meinberlin/views.py:26]: No this was there before, I did a reset a force push to fixup my commits. I think a reset your commit accidentally.

**Comment by der-john** [comment]
works & looks good.

**Pull request merged**